core/Widgets/DateTimePicker/ScheduleButton.vala

```diff
@@ -24,9 +24,9 @@ public class Widgets.ScheduleButton : Gtk.Grid {
     public string label { get; construct; }
 
     private Gtk.Label due_label;
-    
     private Gtk.Box schedule_box;
     private Gtk.Image due_image;
+    private Widgets.DateTimePicker.TimePicker time_picker;
```

Comment:
It is not necessary to import the `Widgets.DateTimePicker.TimePicker`, it is already created in the `Widgets.DateTimePicker.DateTimePicker` widget.

Reply:
I have removed this line.  

Thanks for the info.